frame/system/src/lib.rs

```diff
@@ -975,8 +975,19 @@ where
 	}
 }
 
-/// Ensure that the origin `o` represents an unsigned extrinsic. Returns `Ok` or an `Err` otherwise.
+/// Earlier Usage:
+/// Ensure that the origin `o` represents the root. Returns `Ok` or an `Err` otherwise.
+/// Now Deprecated, Use `ensure_unsigned` instead
```

Comment:
I dont get this comment. The function did not change. Why do you now mention `root`?